Scripts/script-ReadPDFFile.yml

```
@@ -4,12 +4,33 @@ commonfields:
 name: ReadPDFFile
 script: |-
   import pdfx
+  from pdfminer.psparser import PSEOF
   import sys
   reload(sys)
   sys.setdefaultencoding('utf-8')
 
   entryID = demisto.args()['entryID']
 
+  def mark():
```

The name can be more descriptive, for example:
```suggestion
  def mark_suspicious():
```